index.js

```diff
@@ -101,14 +112,5 @@ TypeScript.prototype.cacheKeyProcessString = function(string, relativePath) {
 };
 
 TypeScript.prototype.processString = function (string, relativePath) {
-  try{
-    return ts.transpileModule(string, {compilerOptions: this.options, fileName: relativePath}).outputText;
-  }catch(e){
-    console.error('TYPESCRIPT ERROR:');
-    console.error(e.stack + '\n');
-
-    return '';
-  }
```

Comment:
If there is a compiler error, we want that build attempt to fail and reject. This allows us to display helpful error messages to the user and perform needed cleanup.